lib/tests/unit/connector/connection_test.cc

```diff
@@ -232,7 +255,6 @@ TEST_CASE("Connection::~Connection", "[connection]") {
         // Connector's dtor should be able to effectively stop the
         // event handler and join its thread
         REQUIRE(true);
```

Comment:
This is probably unnecessary; presumably the tests would hang if we didn't get this far.